    /**
     * @return \Throwable
     */

    protected function getException()
    {
        return $this->exception;
    }
